packages/discord.js/typings/index.d.ts

```diff
@@ -2025,6 +2025,12 @@ export class LimitedCollection<Key, Value> extends Collection<Key, Value> {
   public keepOverLimit: ((value: Value, key: Key, collection: this) => boolean) | null;
 }
 
+export interface MessageCall {
+  get endedAt(): Readonly<Date> | null;
```

Comment:
This is technically not a `Readonly<Date>`, is it? I think this should be removed. In JSDoc, it was added because it's a getter.